live/prometheus/main.tf

```diff
@@ -0,0 +1,6 @@
+resource "helm_release" "kube_prometheus_stack" {
```

Comment:
This piece of code is exactly the same as for kafka operator.
Maybe it's a good idea to create a module for helm installation?